examples/nats/nats.go

```diff
@@ -0,0 +1,53 @@
+package nats
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+// natsContainer represents the nats container type used in the module
+type natsContainer struct {
+	testcontainers.Container
+	URI string
+}
+
+// runContainer creates an instance of the nats container type
+func runContainer(ctx context.Context, opts ...testcontainers.ContainerCustomizer) (*natsContainer, error) {
+	req := testcontainers.ContainerRequest{
+		Image:        "nats:latest",
```

Comment:
do we really want to test against latest?

Reply:
ah I usually use latest personally, but I get the rationale. I changed to use nats:2.9